.github/workflows/ci.yml

```
@@ -106,3 +136,35 @@ jobs:
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install locust==2.42.2
+      - name: Start mock service
+        env:
+          MOCK_UPSTREAMS: '1'
+          MOCK_LLM_LATENCY_MS: '50'
+          MOCK_HORIZON_LATENCY_MS: '25'
+          SAFETY_PIPELINE_ENABLED: 'false'
+          AUTH_DISABLED: 'true'
+          GEMINI_API_KEY: ''
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2015:info:2:68: Note that A && B || C is not if-then-else. C may run when A is true

(shellcheck)